web/src/message_edit.ts

```diff
@@ -796,7 +858,7 @@ export function end_message_row_edit($row) {
     if (message !== undefined && currently_editing_messages.has(message.id)) {
         currently_editing_messages.delete(message.id);
         message_lists.current.hide_edit_message($row);
-        compose_call.abort_video_callbacks(message.id);
+        compose_call.abort_video_callbacks(message.id.toString());
```

Comment:
This actually fixes a bug introduced by 9cf9cec43a717b01e3c9d05d162ff9782676583e.

Reply:
Ah nice!

Reply:
It's helpful to split things like this out as independent commits; in this case we probably want to backport that bug fix.

I guess instead perhaps you can just submit a PR to the 9.x branch fixing this line.

Reply:
#31401